Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
```

Comment:
### [timeout](https://github.com/ruby/timeout)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.2.0 → 0.3.0

[_~~change-log~~, [source-code](https://github.com/ruby/timeout), [gem-diff](https://my.diffend.io/gems/timeout/0.2.0/0.3.0)_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/ruby/timeout/compare/v0.2.0...v0.3.0).

These are the first 10 commits:
- (45f3e4d) [add 3.0 to test matrix](https://github.com/ruby/timeout/commit/45f3e4d20b2029ae26d31b9f9ab5ac59425c3a1b)
- (8801539) [CI: use bundler-cache, and 'bundle exec'](https://github.com/ruby/timeout/commit/8801539b041f23dd31dd8aa620e7cc6a815e8722)
- (7238852) [Merge pull request #9 from ruby/ci-use-bundler-cache](https://github.com/ruby/timeout/commit/723885291f27f2c10ed22b2f59a0450a3990bb2d)
- (aed6fff) [Merge pull request #8 from jjb/patch-1](https://github.com/ruby/timeout/commit/aed6fffcbbad3619df44e38d034d9338ba538e68)
- (50c08a9) [s/RubyVM::JIT/RubyVM::MJIT/g](https://github.com/ruby/timeout/commit/50c08a9864fb54c3285068ee66277212eecefb72)
- (7e07629) [Merge pull request #10 from k0kubun/rubyvm-mjit](https://github.com/ruby/timeout/commit/7e0762954ba0d2d48d41b160ce72b718ab9ca6a7)
- (281b52b) [Added dependabot](https://github.com/ruby/timeout/commit/281b52b17e7950197bbbf9c0cb22d957cd2f0d57)
- (a7a5d62) [Merge pull request #12 from ruby/add-dependabot](https://github.com/ruby/timeout/commit/a7a5d62a4554312113988d47a5326763c3355237)
- (ffffb62) [Bump actions/checkout from 2 to 3](https://github.com/ruby/timeout/commit/ffffb621039833ee0524285733d4f475f86749a0)
- (9b18d10) [Merge pull request #13 from ruby/dependabot/github_actions/a…](https://github.com/ruby/timeout/commit/9b18d100e118b3af4db79bab1817ff0dc91c5e42)


</details>